### ISSUE TITLE
我用这个程序会出现nan

### ISSUE THREAD

**Issue by zhangtao22** [zhezh/focalloss#3]
大家有没有类似情况

**Comment by ytwas** [comment]
logits没有 softmax？
